rust/lance/src/io/exec/planner.rs

```diff
@@ -295,8 +296,8 @@ impl Planner {
     fn parse_sql_expr(&self, expr: &SQLExpr) -> Result<Expr> {
         match expr {
             SQLExpr::Identifier(id) => {
-                if id.quote_style == Some('"') {
-                    Ok(Expr::Literal(ScalarValue::Utf8(Some(id.value.clone()))))
+                if id.quote_style == Some('`') {
+                    Ok(Expr::Column(Column::from_name(id.value.clone())))
```

Comment:
Since [our docs state that \` should be used](https://lancedb.github.io/lance/read_and_write.html#filter-push-down) I'd remove `"` as an option.